spec/io/io_spec.rb

```diff
@@ -110,6 +110,16 @@
         @expected = Daru::DataFrame.new({
           :id => id, :name => name, :age => age, :city => city, :a1 => a1
           }, order: [:id, :name, :age, :city, :a1])
+
+        id   = Daru::Vector.new(['id', 1.0, 2.0, 3.0, 4.0, 5.0, 6.0])
```

Comment:
You can try to use already present dataframes in spec file if it is similar. If there is new case or the case where your code snippet was failing (the example you mentioned in PR description), then you can add it too.